src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeInterface.scala

```diff
@@ -62,7 +62,7 @@ object TezosNodeInterface extends TezosRPCInterface with LazyLogging {
           )
         )
       val response: HttpResponse = Await.result(responseFuture, Duration.apply(awaitTimeInSeconds, SECONDS))
-      val responseBodyFuture = response.entity.toStrict(90.second).map(_.data).map(_.utf8String)
+      val responseBodyFuture = response.entity.toStrict(90.second).map(_.data.utf8String)
```

Comment:
Can we make this configurable?

Reply:
the `90 seconds` timeout?

Reply:
That's strange, awaitTimeInSeconds is already there, but that one line has 90 instead of that variable. Replacing 90 with awaitTimeInSeconds should be enough.

Reply:
@ivanopagano Yes

@ba205 `awaitTimeInSeconds` is meant to be for DB operations.

Reply:
@vishakh I will change it with a property of the `application.conf`